go.mod

```diff
@@ -1,16 +1,17 @@
-module github.com/northwesternmutual/grammes
+module github.com/Kaleidoscope-Inc/grammes
 
 go 1.13
 
 require (
 	github.com/google/uuid v1.1.0
-	github.com/gopherjs/gopherjs v0.0.0-20190309154008-847fc94819f9 // indirect
 	github.com/gorilla/websocket v1.4.0
-	github.com/pkg/errors v0.8.1 // indirect
-	github.com/smartystreets/assertions v0.0.0-20190215210624-980c5ac6f3ac // indirect
+	github.com/northwesternmutual/grammes v1.2.0
+	github.com/pkg/errors v0.9.1 // indirect
 	github.com/smartystreets/goconvey v0.0.0-20190306220146-200a235640ff
-	github.com/stretchr/testify v1.3.0 // indirect
-	go.uber.org/atomic v1.3.2 // indirect
-	go.uber.org/multierr v1.1.0 // indirect
+	github.com/stretchr/testify v1.8.1 // indirect
+	go.uber.org/atomic v1.10.0 // indirect
+	go.uber.org/multierr v1.9.0 // indirect
 	go.uber.org/zap v1.9.1
 )
+
+replace github.com/northwesternmutual/grammes v1.2.0 => github.com/Kaleidoscope-Inc/grammes v1.2.1-0.20230221065140-059981b86c6f
```

Comment:
why does this package attempts to refer to an older version of itself?

Reply:
this was an attempt to not change module everywhere since we want it to use our changes but it failed because go has problems with moving on from her ex - i decided to create a separate [branch](https://github.com/Kaleidoscope-Inc/grammes/tree/k6scope-dev) and use this in our code

Reply:
Oh I see that makes sense. That probably has to do with go's package mechanism + forking. Anyway, LGTM!